### DISCUSSION TITLE
[solved ] can't find the extension github copilot

### DISCUSSION THREAD

**Discussion by Racso2006** [community/community#14035] [Copilot Conversations] [answered]
When I am going to manage extensions of my visual studio 2019 and in the search engine I put copilot, the search engine does not find anything, why could it be? 

sorry if it's silly but I can't solve it, and sorry for my bad english 

![2022_04_01_12_29_48_Window](https://user-images.githubusercontent.com/29780779/161247066-04c43eb4-ef2f-46cb-b236-efed2ef0c7a9.png)

**Comment by slpkbt** [comment]
Just update your VS, my friend has same problem so update can fix it

**Comment by slpkbt** [comment, marked as answer]
![изображение](https://user-images.githubusercontent.com/45075347/161251745-913bf1d2-133f-442f-ac41-aa90e5f3ee9a.png)

**Comment by Racso2006** [comment]
perfect thank you very much solved

**Comment by avshyz** [comment]
Had the same problem on arch
To solve it - I've followed the pinned comment [here](https://github.com/orgs/community/discussions/14035), and executed the following command:

```
sudo pacman -S code-features code-features-insiders code-marketplace
```
